pkg/datapath/linux/probes/managed_neighbors.go

```diff
@@ -0,0 +1,84 @@
+package probes
+
+import (
+	"fmt"
+	"net"
+	"runtime"
+
+	"github.com/vishvananda/netlink"
+	"github.com/vishvananda/netns"
+)
+
+// HaveManagedNeighbors creates a netlink.Neigh entry with the flag NTF_EXT_MANAGED.
```

Comment:
This needs to highlight the function's semantics:
```suggestion
// HaveManagedNeighbors returns nil if the host supports managed neighbor entries (NTF_EXT_MANAGED).
// Any errors encountered during the feature probe are returned verbatim.
```